ipfs/chunking.nim

```diff
@@ -7,4 +7,5 @@ proc createObject*(file: File): IpfsObject =
   IpfsObject(data: cast[seq[byte]](contents))
```

Comment:
there's generally `stew.toBytes` - casting between string and seq should be avoided as it results in invalid strings